code/modules/reagents/newchem/drinks.dm

```diff
@@ -62,3 +62,117 @@
 	result = "jackrose"
 	required_reagents = list("applejack" = 4, "lemonjuice" = 1)
 	result_amount = 5
+
+
+// ROBOT ALCOHOL PAST THIS POINT
+// WOOO!
+
+
+/datum/reagent/alkasine //Gloriously copied over from ethanol
```

Comment:
Well, there's a lot of stuff changed in relation to ethanol, not sure what else I should have done here.